src/bare-metal/aps/examples/src/pl011.rs

```diff
@@ -122,10 +122,7 @@ impl Uart {
 
         // SAFETY: We know that self.registers points to the control registers
         // of a PL011 device which is appropriately mapped.
-        unsafe {
-            // Write to the TX buffer.
```

Comment:
I'd like to keep this comment, as it's not immediately obvious what this line does. How about moving it up above the safety comment?